Yoast/wordpress-seo
# [QAK-2179] Fix duplicate queries from multiple admin columns

## Context


* Avoid duplicate queries on the posts overview.

## Summary


This PR can be summarized in the following changelog entry:

* Prevents duplicate queries on the posts overview.

## Test instructions

This PR can be tested by following these steps:

* Go to the posts overview.
* Enable all yoast columns, both for our metadata ( SEO title, meta description etc. ) and the internal link counts.
* There should not be any duplicate queries.
* All column should show expected results.

## UI changes

## Documentation

## Quality assurance


Fixes QAK-2179
